types/react-redux/react-redux-tests.tsx

```diff
@@ -357,22 +357,80 @@ namespace TestTOwnPropsInference {
         return { state: 'string' };
     }
 
-    const ConnectedWithoutOwnProps = connect(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
-    const ConnectedWithOwnProps = connect(mapStateToPropsWithOwnProps)(OwnPropsComponent);
-    const ConnectedWithTypeHint = connect<StateProps, void, OwnProps>(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
+    // `OwnProps` can not be inferred, which is good
+    // const ConnectedWithoutOwnProps = connect(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
+    // Otherwise, this compiles, which is bad.
+    // React.createElement(ConnectedWithoutOwnProps, { anything: 'goes!' });
 
-    // This compiles, which is bad.
-    React.createElement(ConnectedWithoutOwnProps, { anything: 'goes!' });
+    const ConnectedWithOwnProps = connect(mapStateToPropsWithOwnProps)(OwnPropsComponent);
+    const ConnectedWithTypeHint = connect<StateProps, OwnProps>(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
 
     // This compiles, as expected.
     React.createElement(ConnectedWithOwnProps, { own: 'string' });
 
     // This should not compile, which is good.
-    // React.createElement(ConnectedWithOwnProps, { missingOwn: true });
+    // React.createElement(ConnectedWithOwnProps, { anything: 'goes!' });
```

Comment:
Okay, you're right, this name is more-obviously undesirable noise. Can you change the one on line 378 too?